core/src/main/resources/jenkins/model/Messages_de.properties

```diff
@@ -37,28 +37,30 @@ Hudson.NotUsesUTF8ToDecodeURL=\
   Ihr Container verwendet kein UTF-8, um URLs zu dekodieren. Falls Sie Nicht-ASCII-Zeichen \
   in Jobnamen usw. verwenden, kann dies Probleme mit sich bringen. Beachten Sie bitte die Hinweise zu \
   <a href=''https://jenkins.io/redirect/troubleshooting/utf8-url-decoding''>Tomcat i18N</a>).
-Hudson.ReadPermission.Description=\
```

Comment:
Seems so, `Jenkins` actually imports `hudson.model.Messages` and uses that. Crazy.